### ISSUE TITLE
Feature Request: tell user they're banned 

### ISSUE THREAD

**Issue by CluckCluckChicken** [CluckCluckChicken/Magnifier#86]
if someone gets banned, any requests send to api with their token will be blocked, but for the client it will just fail silently, unless they dont have a token and try to log in.

**Comment by gosoccerboy5** [comment]
<s>do what jeffalo did and rickroll them</s>

**Comment by CluckCluckChicken** [comment]
> do what jeffalo did and rickroll them

no i have to go one step further and [wide putin them](https://www.youtube.com/watch?v=Wl959QnD3lM)

**Comment by Quantum-Codes** [comment]
This is why magnifier doesn't work for me???? @CluckCluckChicken  it takes forever to load (loading never stops. Even after 30 mins)

**Comment by CluckCluckChicken** [comment]
> This is why magnifier doesn't work for me???? @CluckCluckChicken it takes forever to load (loading never stops. Even after 30 mins)
> 
> Pls if you didn't do then don't do :(
> Whats the use of this tho?? I am able to access scratch api even when I'm banned.. (I'm able to access the authentication-required API )

you arent banned from magnifier,

**Comment by Quantum-Codes** [comment]
> > This is why magnifier doesn't work for me???? @CluckCluckChicken it takes forever to load (loading never stops. Even after 30 mins)
> > Pls if you didn't do then don't do :(
> > Whats the use of this tho?? I am able to access scratch api even when I'm banned.. (I'm able to access the authentication-required API )
> 
> you arent banned from magnifier,

I am banned on scratch. And magnifier doesn't load for me

**Comment by CluckCluckChicken** [comment]
> > > This is why magnifier doesn't work for me???? @CluckCluckChicken it takes forever to load (loading never stops. Even after 30 mins)
> > > Pls if you didn't do then don't do :(
> > > Whats the use of this tho?? I am able to access scratch api even when I'm banned.. (I'm able to access the authentication-required API )
> > 
> > 
> > you arent banned from magnifier,
> 
> I am banned on scratch. And magnifier doesn't load for me

i don't think that's related. the magnifier frontend doesn't even use any scratch apis.

**Comment by Quantum-Codes** [comment]
> > > > This is why magnifier doesn't work for me???? @CluckCluckChicken it takes forever to load (loading never stops. Even after 30 mins)
> > > > Pls if you didn't do then don't do :(
> > > > Whats the use of this tho?? I am able to access scratch api even when I'm banned.. (I'm able to access the authentication-required API )
> > > 
> > > 
> > > you arent banned from magnifier,
> > 
> > 
> > I am banned on scratch. And magnifier doesn't load for me
> 
> i don't think that's related. the magnifier frontend doesn't even use any scratch apis.

Then why doesn't it load? Should I create an issue?

**Comment by CluckCluckChicken** [comment]
> > > > > This is why magnifier doesn't work for me???? @CluckCluckChicken it takes forever to load (loading never stops. Even after 30 mins)
> > > > > Pls if you didn't do then don't do :(
> > > > > Whats the use of this tho?? I am able to access scratch api even when I'm banned.. (I'm able to access the authentication-required API )
> > > > 
> > > > 
> > > > you arent banned from magnifier,
> > > 
> > > 
> > > I am banned on scratch. And magnifier doesn't load for me
> > 
> > 
> > i don't think that's related. the magnifier frontend doesn't even use any scratch apis.
> 
> Then why doesn't it load? Should I create an issue?
> How long it loads for you?

loading comments takes around 1-5 seconds for me. do you get any console errors?

**Comment by Quantum-Codes** [comment]
> > > > >  -snip-
> > > > > 
> > > > > you arent banned from magnifier,
> > > > 
> > > > 
> > > > I am banned on scratch. And magnifier doesn't load for me
> > > 
> > > 
> > > i don't think that's related. the magnifier frontend doesn't even use any scratch apis.
> > 
> > 
> > Then why doesn't it load? Should I create an issue?
> > How long it loads for you?
> 
> loading comments takes around 1-5 seconds for me. do you get any console errors?

No, I don't... it just simply rotates the logo and shows loading. Thats it.

**Comment by CluckCluckChicken** [comment]
> > > > > > -snip-
> > > > > > you arent banned from magnifier,
> > > > > 
> > > > > 
> > > > > I am banned on scratch. And magnifier doesn't load for me
> > > > 
> > > > 
> > > > i don't think that's related. the magnifier frontend doesn't even use any scratch apis.
> > > 
> > > 
> > > Then why doesn't it load? Should I create an issue?
> > > How long it loads for you?
> > 
> > 
> > loading comments takes around 1-5 seconds for me. do you get any console errors?
> 
> No, It doesn't... it just simply rotates the logo and shows loading. Thats it.

is this loading comments or just accessing magnifier

also users that haven't been indexed by scratchdb don't work.

**Comment by Quantum-Codes** [comment]
> > > > > > > -snip-
> > > > > > > you arent banned from magnifier,
> > > > > > 
> > > > > > 
> > > > > > I am banned on scratch. And magnifier doesn't load for me
> > > > > 
> > > > > 
> > > > > i don't think that's related. the magnifier frontend doesn't even use any scratch apis.
> > > > 
> > > > 
> > > > Then why doesn't it load? Should I create an issue?
> > > > How long it loads for you?
> > > 
> > > 
> > > loading comments takes around 1-5 seconds for me. do you get any console errors?
> > 
> > 
> > No, It doesn't... it just simply rotates the logo and shows loading. Thats it.
> 
> is this loading comments or just accessing magnifier
> 
> also users that haven't been indexed by scratchdb don't work.

I am indexed! I can searchmy data through scratchdb... when i put the url in searchbar(chrome), then it shows white page with Loading... text and the logo rotating. Loading is never complete.

FYI: I was one of the first 50 people who used magnifier.  That day it worked. Not working after that

**Comment by CluckCluckChicken** [comment]
> > > > > > > > -snip-
> > > > > > > > you arent banned from magnifier,
> > > > > > > 
> > > > > > > 
> > > > > > > I am banned on scratch. And magnifier doesn't load for me
> > > > > > 
> > > > > > 
> > > > > > i don't think that's related. the magnifier frontend doesn't even use any scratch apis.
> > > > > 
> > > > > 
> > > > > Then why doesn't it load? Should I create an issue?
> > > > > How long it loads for you?
> > > > 
> > > > 
> > > > loading comments takes around 1-5 seconds for me. do you get any console errors?
> > > 
> > > 
> > > No, It doesn't... it just simply rotates the logo and shows loading. Thats it.
> > 
> > 
> > is this loading comments or just accessing magnifier
> > also users that haven't been indexed by scratchdb don't work.
> 
> I am indexed! I can searchmy data through scratchdb... when i put the url in searchbar(chrome), then it shows white page with Loading... text and the logo rotating. Loading is never complete.
> 
> FYI: I was one of the first 50 people who used magnifier. That day it worked. Not working after that

your user page loads for me:
![magnifier quantum codes](https://user-images.githubusercontent.com/46281840/125037274-d3989400-e08b-11eb-8007-ce3dbbbdb8ff.png)

**Comment by Quantum-Codes** [comment]
@CluckCluckChicken  i am not even able to load **home page..**  thats my problem

**Comment by Quantum-Codes** [comment]
![20210709_125253](https://user-images.githubusercontent.com/87054411/125039900-a44a4d00-e0b4-11eb-844a-ed84ace51a67.jpg)
This is where I'm stuck

**Comment by CluckCluckChicken** [comment]
that's a different issue. what browser are you using?

**Comment by Quantum-Codes** [comment]
> that's a different issue. what browser are you using?

Chrome

**Comment by Quantum-Codes** [comment]
https://github.com/CluckCluckChicken/Magnifier/issues/112
